integration/test-integration/src/test/resources/openAPIs/app_level_security_openAPI.yaml

```diff
@@ -29,10 +29,6 @@ paths:
             $ref: '#/definitions/Pet'
         '404':
           description: Pet not found
-      x-wso2-application-security:
```

Comment:
APIM by default adds this to the openAPI definition. Wouldn't we need this to simulate that behavior?

Reply:
we now only reads security given in the api.yaml. we don't need to use this extension anymore as it adds more unnecessary complexity so choreo-connect doesn't support this extension.

Reply:
Previously we obtained APIM enabled API keys using the above extension. 
https://github.com/wso2/product-microgateway/blob/main/adapter/internal/oasparser/model/constants.go#L42
Still we are using the above extension in constants. Are we going to remove that ?